lib/recurly/element/card-cvv.js

```diff
@@ -0,0 +1,10 @@
+import Element from './element';
+
+export function factory (options) {
+  return new CardCvvElement(Object.assign({}, options, { elements: this }));
```

Comment:
I think object spread syntax might be more readable for situations like this, but that's definitely like just my opinion man and nit picky

`return new CardCvvElement({ ...options, elements: this })`

Reply:
Sure, I prefer the spread syntax as well. `Object.assign` is much faster, but that hardly matters here.